jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/ErrorHandler.java

```diff
@@ -92,6 +92,25 @@ public boolean errorPageForMethod(String method)
         return ERROR_METHODS.contains(method);
     }
 
+    /**
+     * Write an error response.
+     * <p>
+     * In Servlet implementations of the {@link ErrorHandler}, this method is overridden to signal that
+     * a sendError should be triggered a when it enters the servlet channel.
+     * <p>
+     *
+     * @param request  The request.
+     * @param response The response.
+     * @param callback The callback to call when the response is written.
+     * @param code     The error status code.
+     * @return True if the error response was written.
```

Comment:
```suggestion
     * @return {@code True} if the error response was written; {@code False} if the state of the request and/or response has been changed so that by continuing normal handling an error response will be generated by a down stream handler.
```